.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -49,37 +49,34 @@
       - Run `./build.sh` to begin clone and patch application and see what, if anything, needs to be regenerated.  Issue to track re-patching workflow: [#1468](https://github.com/dotnet/source-build/issues/1468)
-      - [3.1] ASP.NET will always need to be regenerated because it contains a version number that needs to be updated.  The version should be updated to the SDK version being built.  See https://github.com/dotnet/source-build/blob/release/3.1/patches/aspnetcore/0007-Fix-version-number.patch
-1.  - [ ] [Internal] [3.1] Update internal feed list in `.\build.sh`
-      - Internal feeds are listed under `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS`
-      - Make sure to keep `dotnet3.1-internal-transport` in the list
-      - Issue tracking this: [#1674](https://github.com/dotnet/source-build/issues/1674)
+      - [3.1] ASP.NET will always need to be regenerated because it contains a version number that needs to be updated.  The version should be updated to the SDK version being built.  See https://github.com/dotnet/source-build/blob/release/3.1/patches/aspnetcore/0006-Fix-version-number.patch
 1.  - [ ] Start building locally (for quick diagnosis) and in CI (for super clean build) and iterate towards a green build.
       - [Internal] [3.1] Set environment variable `internalPackageFeedPat` to dnceng internal PAT with package read and code read access.
         - E.g. run `export internalPackageFeedPat;read internalPackageFeedPat` and paste the PAT. This sets up your running shell for an authenticated build. Passing `/p:` isn't sufficient.
-      - [Internal] [3.1] Also pass arg `/p:AzDoPat=$internalPackageFeedPat` for darc clone in particular.
-      - [Internal] [2.1] Pass auth arg `/p:ProdConBlobFeedUrlPrefix=<url up to and not including 'orchestrated-release-2-1/' from feed.txt>`
-      - [Internal] [2.1] When running smoke-test locally, also pass `/p:ProdConBlobFeedUrlPrefix=<...>`
-        - The smoke-test command looks like `./build.sh /t:RunSmokeTest /p:Configuration=Release /p:ProdConBlobFeedUrlPrefix=***`.
-      - Update baseline from the CI job, using the `centos71 Offline` artifacts.
-        - `Logs centos71 Offline/source-build/logs/artifacts/prebuilt-report/generated-new-baseline.xml`
-          - => `tools-local/prebuilt-baseline-online.xml`
-        - `Logs centos71 Offline/tarball/logs/artifacts/prebuilt-report/generated-new-baseline.xml`
-          - => `tools-local/prebuilt-baseline-offline.xml`
+      - [Internal] [3.1] Also pass arg `/p:AzDoPat=$internalPackageFeedPat` to build.sh.
+      - [Internal] To run CI, from [dnceng/internal/source-build-pre-arcade-CI](https://dev.azure.com/dnceng/internal/_build?definitionId=251), click "Run pipeline" and select your dev/… branch under "Branch/tag", then click "Run"
+         - If you are unsure if your build will be successful, you can run only the centos71_* stages to save resources.
 1.  - [ ] Review prebuilt baseline diff and iterate builds to drive to zero.
```

Comment:
Where do you find the prebuilt report?  We should provide context as to what a prebuilt is and how you drive to zero.

Reply:
Added some details on this.